docs/reference/requirement-sets/excel-api-requirement-sets.md

```diff
@@ -31,6 +31,420 @@ For more information about versions, build numbers, and Office Online Server, se
 - [Where you can find the version and build number for an Office 365 client application](https://support.office.com/article/version-and-build-numbers-of-update-channel-releases-ae942449-1fca-4484-898b-a933ea23def7)
 - [Office Online Server overview](https://docs.microsoft.com/officeonlineserver/office-online-server-overview)
 
+## What's in the Excel JavaScript API Preview
+
```

Comment:
A few general comments:

- The "Beta" row of the requirement set table at the top of this article directs folks to [GitHub](https://github.com/OfficeDev/office-js-docs/tree/ExcelJs_OpenSpec) -- "Please visit our Excel JavaScript API open specification page!"   
    - Should this messaging in the requirement sets table be replaced with a pointer to the new "What's new in the Excel JavaScript API Preview" section instead? Or is there still value in sending them off to OfficeDev/office-js?

- The [!NOTE] near the top of the article and then the first entry of the requirement set table near the top of the article both mention the "Beta" requirement set | APIs -- and then this section is titled "What's in the Excel JavaScript API Preview".  If "Beta" and "Preview" are the same thing, the wording should be consistent throughout the article (i.e., use either "Beta" OR "Preview" throughout the article).

- Consider moving the first three sentences from the [!NOTE] about Beta (preview) APIs near the top of the article into this new section ("What's in the Excel JavaScript API Preview"), immediately following the section heading..and rewording as appropriate to be consistent with usage of either "Beta" or "Preview".  Maybe something along the lines of:
    - Preview APIs are subject to change and are not intended for use in a production environment. We recommend that you try them out in test and development environments only. Do not use Preview APIs in a production environment or within business-critical documents.

- Consider adding a sentence or two after this section heading and before the [!NOTE] (if you add one) that describes at a high level what "preview APIs" are.  i.e., all new APIs are first made available in Preview, and then subsequently become part of a specific (numbered) requirement set.